distribution/sc-proxy-dex/src/proxy_common.rs

```diff
@@ -155,4 +210,25 @@ pub trait ProxyCommonModule {
     #[view(getLockedAssetTokenId)]
     #[storage_mapper("locked_asset_token_id")]
     fn locked_asset_token_id(&self) -> SingleValueMapper<Self::Storage, TokenIdentifier>;
+
+    #[view(getWrappedLpTokenId)]
+    #[storage_mapper("wrapped_lp_token_id")]
+    fn wrapped_lp_token_id(&self) -> SingleValueMapper<Self::Storage, TokenIdentifier>;
+
+    #[storage_mapper("wrapped_tp_token_nonce")]
+    fn wrapped_lp_token_nonce(&self) -> SingleValueMapper<Self::Storage, Nonce>;
+
+    #[view(getWrappedFarmTokenId)]
+    #[storage_mapper("wrapped_farm_token_id")]
+    fn wrapped_farm_token_id(&self) -> SingleValueMapper<Self::Storage, TokenIdentifier>;
+
+    #[storage_mapper("wrapped_farm_token_nonce")]
+    fn wrapped_farm_token_nonce(&self) -> SingleValueMapper<Self::Storage, Nonce>;
+
+    #[storage_mapper("locked_asset_factory_address")]
+    fn locked_asset_factory_address(&self) -> SingleValueMapper<Self::Storage, Address>;
+
+    #[view(getIntermediatedFarms)]
```

Comment:
You might want to add intermediated pairs as well for consistency

Reply:
done